book/basics.qmd

```diff
@@ -17,22 +17,35 @@ abstract: TODO (150-200 WORDS)
 
 {{< include _setup.qmd >}}
 
-In this chapter, we will introduce the essential building blocks of `r mlr3`, along with the corresponding `r ref_pkg("R6")` classes and operations used for machine learning.
-
-The data, which `r mlr3` encapsulates in [tasks](#sec-tasks), is split into non-overlapping training and test sets.
-As we are interested in models that generalize beyond the training data rather than just memorizing it, separate test data allows to evaluate models in an unbiased way and assess to what extent they have learned the concepts that underlie the data.
-The training data is given to a [learner](#sec-learners), which builds a model based on it.
+In essence, machine learning means learning relationships from data.
+In supervised learning, data consists of labeled examples, i.e., each data point contains feature variables and an associated label (target variable).
+As an example, suppose we want to predict the Miles/(US) gallon a car can drive based on features such as its horsepower and the number of gears.
+In `r mlr3`, data is stored in [tasks](#sec-tasks) that augment a `data.table` with information about the target and features.
```

Comment:
This doesn't feel very natural here, you've kind of jumped from describing ML to randomly putting in one mlr3 point. I don't think the sentence is needed tbh, if you do want to keep it to link with mlr3 terminology then would suggest something with a cleaner link like "In this book, you'll often see us refer to datasets as tasks, which is because predictive problems, like the example above, can be thought of as tasks we're trying to solve for a given dataset".